dxr/plugins/rust/refs.py

```diff
@@ -126,6 +126,27 @@ def menu_items(self):
         return menu
 
 
+class MacroRef(_RustRef):
+    def prepare_menu_data(self, tree_index, datum):
+        self.hover = datum['qualname']
+        return trim_dict(datum, ['qualname'])
+
+    def menu_items(self):
```

Comment:
I'm not quite following - that shouldn't have any impact on macros.

Unless you mean you want to be able to jump to macro references from a macro reference?